examples/google-signin/google-signin-iframe.html

```diff
@@ -0,0 +1,25 @@
+<body>
+  <!--
+    Specify a Google Sign-In OAuth Client ID,
+    then load Google Sign-In's script.
+    https://developers.google.com/identity/sign-in/web/sign-in
+  -->
+  <meta
+    name="google-signin-client_id"
+    content="520465458218-e9vp957krfk2r0i4ejeh6aklqm7c25p4.apps.googleusercontent.com"
+  />
+  <script src="https://apis.google.com/js/platform.js" async defer></script>
```

Comment:
WDYT about moving loading the GAPI script into the gaa-util.js file itself? It makes it impossible for devs to miss loading in the script when they create their own iframe.

Reply:
Keeping it separate means the GAPI JS loads faster. Also it matches the GSI docs:
https://developers.google.com/identity/sign-in/web/sign-in